src/inference/src/dev/core_impl.cpp

```diff
@@ -756,7 +760,7 @@ ov::AnyMap ov::CoreImpl::get_supported_property(const std::string& full_device_n
         ov::hint::allow_auto_batching.name(),
     };
 
-    const auto flattened = ov::parseDeviceNameIntoConfig(full_device_name, user_properties);
+    const auto flattened = ov::parseDeviceNameIntoConfig(full_device_name, user_properties, true);
```

Comment:
Why do we need to introduce an additional parameter?
Get supported property can be called not only "query_from_virtual". Why is it always true, is it correct?

Reply:
as we aligned before, AB related keys should be reported as supported when call get supported property from hetero or auto, like: 
/* hetero/auto codes****/
// query core for supported propery list for its device priority lists
GetCore()->get_supported_property("GPU.0", {ov::hint::allow_auto_batching(false), *other properties*}})
with this call, we expect core to report back batching as supported, and later let core to handle this key during compile models. But here parseDeviceNameIntoConfig will wipe out AB keys.

while parseDeviceNameIntoConfig is been called in many places in core, get supported property currently only been used in hetero and auto, that's why I made changes here.  in what situation does other hardware plugin also need to call this API?

Reply:
@songbell Why we can't explicitly add batch related properties to config where it is needed? I mean in place where virtual plugin calls `GetSupportedConfig`

Reply:
@nkogteva per our previous alignment, core is responsible for reporting the core level properties(batching + cache+..), and these properties should be invisible to plugins

Reply:
> @nkogteva per our previous alignment, core is responsible for reporting the core level properties(batching + cache+..), and these properties should be invisible to plugins

From my point of view, if we need special flag in common function, it means that properties are not invisible. But as far as I understood, if plugin will request supported properties for BATCH they will be kept. Maybe I missing something.

Reply:
@nkogteva the batch properties are invisible to plugins, but not totally invisible in core, because they are core level prooperties. So, for this issue, we either add a flag as in this PR, or we move the clean batch operations out of parseDeviceNameIntoConfig, but in this case, we will need to call clean batch configs after checking batch done in each compile model overloading call. what do you think?